packages/ndla-ui/src/NDLAFilm/FilmContentCardTags.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { spacing, colors, fonts, misc } from '@ndla/core';
+import { css } from '@emotion/core';
+
+const FilmContentCardTags = ({ movieResourceTypes, resourceTypes }) => {
+  const resources = {};
+  movieResourceTypes.forEach(movieResourceType => {
+    const resource = resourceTypes.find(
+      resourceType => resourceType.id === movieResourceType.id,
+    );
+    if (resource) {
+      resources[resource.name] = true;
+    }
+  });
+  return (
+    <div css={wrapperCSS}>
+      {Object.keys(resources).map(resourceName => (
+        <span css={movieTagsCSS} key={resourceName}>
+          {resourceName}
+        </span>
+      ))}
+    </div>
+  );
+};
+
+const wrapperCSS = css`
```

Comment:
Hvorfor kan ikke dette være en styled komponent?

Reply:
Jeg mener det er enklere å lese av DOM'en fra render funksjonen hvis man bruker css`` og HTML-tag istedet for styled komponent.. Men er åpen for å endre hvis du har en god grunn:)?

Reply:
Jeg syntes selv lesbarheten blir bedre med styled. I tillegg så er det jo fint med litt konsistens i koden vi skriver. 

Når det gjelder tekniske grunner kan jeg nå bare se et eksempel der det faktisk lønner seg med styled fremfor css. Ta dette eksemplet hvor jeg sender inn `isActive` som en prop:
```js
const StyledSomething = styled.div`
  color: {p => p.isActive ? '#eee' : '#fff'}
`;

<StyledSomething isActive>
  SomeText
</StyledSomething>
```

For å få samme effekt med css, må vi lage en funskjon slik:
Jeg syntes selv lesbarheten blir bedre med styled. I tillegg så er det jo fint med litt konsistens i koden vi skriver. 

Når det gjelder tekniske grunner kan jeg nå bare se et eksempel der det faktisk lønner seg med styled fremfor css. Ta dette eksemplet hvor jeg sender inn `isActive` som en prop:
```js
const divStyle = (isActive) => css`
  color: {isActive ? '#eee' : '#fff'}
`;

<div css={divStyle(true)}>
  SomeText
</div>
```

Komponenter skal være så små som mulig (uten at det blir alt for snevert) slik at man lett kan se hva for eksempel `StyledSomething` er.

Reply:
Ok, endret den:) Litt usikker på om jeg er enig med om det er mer leselig, men kjøper props argumentasjonen:P